command/sub8_launch/launch/subsystems/blueview.launch

```diff
@@ -10,9 +10,11 @@
             color:
                 enable: True
             # How often to send output (raw data / images), seconds
-            period_seconds: 0.1
+            period_seconds: -1.0
```

Comment:
When the period is <= 0, the driver just produces pings as fast as the driver can, instead of at a fixed interval. The fixed interval thing doesn't even really make sense anyways for this small a number beacause the sonar get only produce a few per second.